src/Symfony/Component/Security/Http/Logout/CookieClearingLogoutHandler.php

```diff
@@ -35,11 +35,11 @@ public function __construct(array $cookies)
     /**
      * Implementation for the LogoutHandlerInterface. Deletes all requested cookies.
      *
-     * @param Request        $request
-     * @param Response       $response
-     * @param TokenInterface $token
+     * @param Request             $request
+     * @param Response            $response
+     * @param TokenInterface|null $token
      */
-    public function logout(Request $request, Response $response, TokenInterface $token)
+    public function logout(Request $request, Response $response, TokenInterface $token = null)
```

Comment:
Changing the signature here is a BC break, same for all other.

Reply:
Right, for applications implementing their own logout handler using `$token` in a case it is not set. I doubt anyone is concerned and I don't want to wait for Symfony flex (we won't be able to use it anyways) for a bug filed 4 years ago.

Reply:
My mistake I didn't realize updating the interface force every implementation to update its signature. So is there any way to fix this before Symfony flex?